pkg/controllers/update.go

```diff
@@ -51,6 +51,9 @@ func RunInstallScript() (bool, string, Error) {
 
 	// write script to temp file
 	tmpFile, err := utils.WriteTempFile("install.sh", script, 0555)
+	if err != nil {
```

Comment:
commits: `Check utils.WriteTempFile error` commit should be prefixed with `chore:`